components/automate-ui/src/app/entities/layout/layout.model.ts

```diff
@@ -1,21 +1,38 @@
+import { Observable } from 'rxjs';
+
+// Sidebar {
+//   active: "dashboards",
+//   dashboards: []
+// }
+
+export interface Sidebars {
+  active?: string;
+  dashboards?: MenuItemGroup[];
+  applications?: MenuItemGroup[];
+  infrastructure?: MenuItemGroup[];
+  compliance?: MenuItemGroup[];
+  settings?: MenuItemGroup[];
+  profile?: MenuItemGroup[];
+}
+
 // MenuItemGroup {
 //   name: "Node Management",
 //   items: [],
-//   visible: true
+//   visible: Observable
 // }
 
 export interface MenuItemGroup {
   name: string;
   items: MenuItem[];
-  visible: any; // boolean or observable
+  visible?: any;
```

Comment:
Hmm... looks like you are still using this as a boolean **or** an Observable, but that makes it very difficult to maintain. Should probably be of type `Observable<boolean>`, just like the `MenuItem` property.

Reply:
@msorens thanks for the catch, it should always be an observable! 👍🏻